### PR TITLE
exposing custom fields

### PR DESCRIPTION
**Pull request by thitu**

### CODE REVIEW

**Comment by thitu** [comment]
modified certificate expires in 3 years

**Comment by thitu** [comment]
fyi...
```Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            85:c3:a9:ed:b9:0e:91:45
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, ST=AZ, L=Scottsdale, O=Trusona, CN=trusona.com/emailAddress=engineering@trusona.com
        Validity
            Not Before: Jul 10 17:04:48 2019 GMT
            Not After : Jul 10 17:04:48 2020 GMT
        Subject: C=US, ST=AZ, L=Scottsdale, O=Trusona, CN=trusona.com/emailAddress=engineering@trusona.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
```

File: lib/trusona/trusonafication.rb [hunk 1]
```diff
@@ -58,6 +58,9 @@ def self.find(trusonafication_id)
     # @param timeout [Int] (30) The max amount of time, in seconds, to wait
     #  for a response from the Trusona API when polling for a Trusonafication
     #  result
+    # @option params [Hash] :custom_fields Optional data to be associated with
+    #  this trusonafication and can be used to constomize any UX elements. Total
```
Comment (jmtrusona) [inline lib/trusona/trusonafication.rb]: Looks like the word "trusonafication" is capitalized in the lines above and below. This should probably be capitalized for consistency.

Comment (thitu) [reply lib/trusona/trusonafication.rb]: 👍 updated

**Pull request merged**